src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContractSerializer.cs

```diff
@@ -66,6 +66,8 @@ public DataContractSerializer(Type type, XmlDictionaryString rootName, XmlDictio
 
 #if NET_NATIVE
         public DataContractSerializer(Type type, IEnumerable<Type> knownTypes, int maxItemsInObjectGraph, bool ignoreExtensionDataObject, bool preserveObjectReferences)
+#elif MERGE_DCJS
+        internal DataContractSerializer(Type type, IEnumerable<Type> knownTypes, int maxItemsInObjectGraph, bool ignoreExtensionDataObject, bool preserveObjectReferences)
```

Comment:
This "elif" doesn't seem right to me. Should the "endif" be right after line 70?